packages/react-liveness/src/components/FaceLivenessDetector/service/utils/videoRecorder.ts

```diff
@@ -1,3 +1,7 @@
+import { isAndroidChromeWithBrokenH264 } from '../../utils/device';
+
+const ALTERNATE_MIME_TYPE = 'video/x-matroska;codecs=vp8';
```

Comment:
Can we add a small comment explaining that this should only be used in chrome and maybe change the name to `ALTERNATE_CHROME_MIME_TYPE`?